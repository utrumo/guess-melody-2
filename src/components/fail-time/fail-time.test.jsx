import React from 'react';
import renderer from 'react-test-renderer';

import FailTime from './fail-time.jsx';

it(`FailTIme renders correctly`, () => {
  const tree = renderer
    .create(<FailTime onButtonClick={jest.fn()}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
