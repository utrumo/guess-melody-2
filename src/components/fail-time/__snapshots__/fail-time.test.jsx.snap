// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FailTIme renders correctly 1`] = `
<section
  className="result"
>
  <div
    className="result__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="result__title"
  >
    Увы и ах!
  </h2>
  <p
    className="result__total result__total--fail"
  >
    Время вышло! Вы не успели отгадать все мелодии
  </p>
  <button
    className="replay"
    onClick={[MockFunction]}
    type="button"
  >
    Попробовать ещё раз
  </button>
</section>
`;
